cmd/maya-exporter/app/command/command.go

```diff
@@ -46,10 +46,10 @@ func NewCmdVolumeExporter() (*cobra.Command, error) {
 	options.MetricsPath = "/metrics"
 	cmd := &cobra.Command{
 		Short: "Collect metrics from OpenEBS volumes",
-		Long: `  maya-volume-exporter monitors openebs volumes and exporter the metrics.
-  It starts collecting metrics from the jiva volume controller at the endpoint
-  "/v1/stats" Prometheus Server can collect the metrics from maya-volume-exporter.`,
-		Example: `  maya-volume-exporter -a=http://localhost:8001 -c=:9500 -m=/metrics`,
+		Long: `maya-exporter monitors openebs volumes and export the metrics.
+It starts collecting metrics from the jiva volume controller at the endpoint
```

Comment:
maya-exporter can be deployed alongside the openebs volume or pool containers as side cars.